Source/WebCore/Modules/mediastream/gstreamer/GStreamerStatsCollector.cpp

```diff
@@ -160,6 +158,11 @@ static inline void fillInboundRTPStreamStats(RTCStatsReport::InboundRtpStreamSta
     if (gst_structure_get_uint64(structure, "bytes-received", &bytesReceived))
         stats.bytesReceived = bytesReceived;
 
+    if (additionalStats) {
+        if (gst_structure_get_uint64(additionalStats, "frames-decoded", &value))
```

Comment:
Maybe you can trust the short circuit evaluation and put everything in the same line with an &&